### ISSUE TITLE
Surround AL0432 with Pragma --> Add option (Review Later)

### ISSUE THREAD

**Issue by fvet** [BartPermentier/al-toolbox#46]
Following https://github.com/HodorNV/ALOps/issues/381, I'm still looking for a decent solution to show / treat / delay AL0432 warnings as part of our pipelines.

- AL0432 warnings are interesting to see in any BC / ISV **nextMinor / nextMajor** pipelines, to verify the impact of new BC versions on our ISV apps / new ISV versions on our PTE apps. 
- AL0432 warnings may however not fail my **current** pipelines (with 'fail_on_warnings' enabled) Demoting AL0432 in a ruleset to **Info** might be a possible solution, but then it has an impact again on the nextMinor / nextMajor (no longer showing the warnings). Having 2 separate rulesets is also cumbersome to maintain...

Conclusion: we currently have the AL0432 rule set with action **Warning** and thus these warnings require attention (1/2).

Next, we've obsoleted some of our own ISV tables (obsoletestate = Pending). For some features, we do not support current / new behavior side-by-side and thus also obsoleted some procedures / publishers, (in fact, 'obsoleted' means 'removed / no longer supported' in these scenarios) resulting in AL0432 warnings too.

(1) These warnings can be **ignored** (forever - untill we can actually delete the procedures / publishers) using the 'Surround AL0432 with Pragma', which adds the pragma directive 

`#pragma warning disable AL0432`

(2) However, we'd like to treat the other **BC** AL0432 warnings **that require future attention** by setting a temporary pragma, **to be removed / treated in a later release**. Current idea is to mark them with a // TODO tag.

`#pragma warning disable AL0432 // TODO - Review in next release`

![image](https://user-images.githubusercontent.com/25268332/125352329-6887df80-e361-11eb-8435-327818674be5.png)

Not sure if this make sense, but we couldn't find a better way.

**Long story short:** 
Would it be possible for AL Toolbox to add a 'Surround AL0432 with Pragma **(// TODO)**' or 'Surround AL0432 with Pragma **(Review Later)**' option? It at least points the developer to make a choice on how to set the pragma.

**Comment by BartPermentier** [comment]
Hello Frédéric,

Seems like a usefull addition, I've added it to the queue. But mind that it can take a few weeks / months as we only develop for this project if we got time to spare.
You can speed up the process if you have your own JScript Developer that can do the requested change then I'll merge it :)

**Comment by fvet** [comment]
There's a good developer coming your way ;)

**Comment by BartPermentier** [comment]
Well i'll assign this as his first task then, so the sooner you let him go... ;)

**Comment by dsaveyn** [comment]
Hi Frédéric,

We solved this one for you. A new quick action was introduced which allows you to add a pragma with a TODO tag and an optional comment. This will be made available in the next release.

Here's a sneak preview:

![Pragma With Comment](https://user-images.githubusercontent.com/92365904/137319387-f5042d66-4576-48bc-96ce-5d0c5be4f0a0.gif)

@BartPermentier FYI
